docs/hacker-stats-intro/utils.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def ECDF(data):
+    x = np.sort(data)
+    y = np.cumsum(x) / np.sum(x)
+    
+    return x, y
+
+
+def despine(ax):
+    ax.spines['right'].set_visible(False)
+    ax.spines['top'].set_visible(False)
```

Comment:
With the `matplotlibrc` file in place, do we need this function? If so, we keep it.

Reply:
I don't think I wrote any of this 👯‍♂️